docs/index.md

```diff
@@ -15,9 +15,12 @@ It allows you to make queries using the powerful [SQLAlchemy Core][sqlalchemy-co
 expression language, and provides support for PostgreSQL, MySQL, and SQLite.
 
 Databases is suitable for integrating against any async Web framework, such as [Starlette][starlette],
-[Sanic][sanic], [Responder][responder], [Quart][quart], [aiohttp][aiohttp], [Tornado][tornado], or [FastAPI][fastapi].
+[Sanic][sanic], [Responder][responder], [Quart][quart], [aiohttp][aiohttp], [Tornado][tornado],
+[FastAPI][fastapi] or [Esmerald][esmerald].
 
-**Requirements**: Python 3.7+
+**Documentation**: [https://www.encode.io/databases/](https://www.encode.io/databases/)
```

Comment:
You shouldn't use full path.

Reply:
This was my VSCode. Not even sure how it was done. Ok, let me have a look at everything